extensions/webview/init.lua

```diff
@@ -17,14 +17,45 @@
 ---
 --- This module provides support for injecting custom JavaScript user content into your webviews and for JavaScript to post messages back to Hammerspoon.
 
-local module       = require("hs.webview.internal")
-module.usercontent = require("hs.webview.usercontent") ;
+local USERDATA_TAG = "hs.webview"
+
+local osVersion = require"hs.host".operatingSystemVersion()
```

Comment:
to me it scans better than including parenthesis or adding a space when you're really only going for one function or table from the module... and as no one's published style requirements for HS... but I'll change them if anyone cares enough